src/smt/solver_engine.cpp

```diff
@@ -353,8 +353,7 @@ void SolverEngine::setInfo(const std::string& key, const std::string& value)
     d_env->getStatisticsRegistry().registerValue<std::string>(
         "driver::filename", value);
   }
-  else if (key == "smt-lib-version"
-           && !getOptions().base.inputLanguageWasSetByUser)
+  else if (key == "smt-lib-version")
```

Comment:
Just to make sure this is intended: set-info always has preference over user provided options?